cob_hardware_config/robots/cob4-7/urdf/cob4-7.urdf.xacro

```diff
@@ -15,6 +15,12 @@
   <!-- torso -->
   <xacro:include filename="$(find cob_description)/urdf/cob4_torso/torso.urdf.xacro" />
 
+  <!-- arm -->
+  <xacro:include filename="$(find cob_description)/urdf/cob4_arm/cob4_arm.urdf.xacro" />
+
+  <!-- hand -->
+  <xacro:include filename="$(find cob_description)/urdf/cob4_gripper/gripper.urdf.xacro" />
```

Comment:
grippers are not added to the urdf...and also not part of the robot.xml...is that intended?

Reply:
the motors of the hands are missed/broken... I am not sure if the hands will be replaced,  still to be clarified...